packages/email/email.js

```diff
@@ -116,11 +119,11 @@ const getTransport = function() {
   if (
     this.cacheKey === undefined ||
     this.cacheKey !== url ||
-    this.cacheKey !== packageSettings?.service ||
+    this.cacheKey !== packageSettings.service ||
```

Comment:
why this removal?

Reply:
It can't be [null](https://github.com/meteor/meteor/pull/11891/files#diff-81b3d1c2cbdddcb39f637030fdb2c2fe5e3190c2b3e0b638c68199f793aa7e5aR114)